ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/page/content/clientlibs/js/paths.js

```diff
@@ -35,5 +35,12 @@ $(function () {
                 });
             });
         }
+        var switchField = $('#create-package-switch')[0];
+        $(switchField).on('change', function(e){
+            $('#create-package-multifield foundation-autocomplete').each(function (){
+                let isChecked = switchField.hasAttribute('checked');
```

Comment:
Please note: for reasons historical, the ES6 syntax is not used in this project (this is justified by the issues that may arize with a legacy code minifier in an AEM instance. If I remember it correctly, the issue is not quite the issue anymore :) but this project's code is what it is.
And if we did use ES6 syntax, this would be `const` not `let` because the variable is effectively immutable